governance/board-sync.py

```
@@ -0,0 +1,354 @@
+# ---------- 期望状态（state 全集与颜色唯一来源：expected-state.json#labels.items） ----------
+
+def load_states():
+    try:
+        with open(os.path.join(DIR, "expected-state.json"), encoding="utf-8") as f:
+            items = json.load(f)["labels"]["items"]
+    except Exception as e:
+        raise Infra(f"expected-state.json 读取失败: {e}") from e
+    states = [{"name": it["name"][len("state:"):], "color": "#" + it["color"]}
+              for it in items if it.get("name", "").startswith("state:")]
+    if not states:
+        raise Infra("expected-state.json 无 state:* 标签——期望状态缺失（fail-closed，同 drift §16）")
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)